src/lib/input/input.md

```diff
@@ -129,18 +129,18 @@ the error messages.
 function myErrorStateMatcher(control: FormControl, form: FormGroupDirective | NgForm): boolean {
   // Error when invalid control is dirty, touched, or submitted
   const isSubmitted = form && form.submitted;
-  return !!(control.invalid && (control.dirty || control.touched || isSubmitted)));
+  return !!(control.invalid && (control.dirty || control.touched || isSubmitted));
```

Comment:
Yeah it is unnecessary, but it's based on the default errorStateMatcher which originates from https://github.com/angular/material2/pull/3560#discussion_r106483976.

Happy to change it if coercion is unnecessary, but it does add a little type safety. A little more discussion in https://github.com/angular/material2/issues/6585